lib/surface/components/form/range_input.ex

```diff
@@ -0,0 +1,62 @@
+defmodule Surface.Components.Form.RangeInput do
+  @moduledoc """
+  Generates a text input.
+
+  Provides a wrapper for Phoenix.HTML.Form's `range_input/3` function.
+
+  All options passed via `opts` will be sent to `range_input/3`, `value` and
+  `class` can be set directly and will override anything in `opts`.
+
+
+  ## Examples
+
+  ```
+  <RangeInput form="volume" field="percent" opts={{ [min: "0", max: "100"] }}>
```

Comment:
```suggestion
  <RangeInput form="volume" field="percent" min="0" max="100" />
```

Since you've added the min and max as explicit properties, maybe the example could reflect that.

Reply:
The first attempt was to pass `min` and `max` to options. Thanks